TabulateSmarterTestAdminPackage.Common/Validators/Validator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using TabulateSmarterTestAdminPackage.Common.Enums;
+
+namespace TabulateSmarterTestAdminPackage.Common.Validators
+{
+    public abstract class Validator
+    {
+        protected Validator(ErrorSeverity errorSeverity, object parameter)
```

Comment:
I see that some Validators like `RequiredStringValidator` pass in `null` since they don't have anything relevant.  How about we make that parameter optional so it doesn't have to be initialized with null?

`protected Validator(ErrorSeverity errorSeverity, object parameter = null)`